virt/virt-web-console-overview.adoc

```diff
@@ -771,6 +771,9 @@ You can manage disks on the *Disks* tab.
 |Search field
 |Search for a disk by name.
 
+|*Mount Windows driver disk* checkbox
+|Mounts an ephemeral container disk as a CD-ROM
```

Comment:
I would add a period since it's a sentence (it looks like the other list items follow that convention).

Reply:
I think it's a sentence fragment. However, I can rewrite this as "Select to mount ...etc."